test/eth/Web3Service.spec.js

```diff
@@ -219,7 +221,8 @@ test('connect to ganache testnet with account 0x16fb9...', done => {
     .connect()
     .then(() => service.eth.getAccounts())
     .then(accounts => {
-      expect(accounts.slice(0, 2)).toEqual(expectedAccounts);
+      expect(accounts[0].toLowerCase()).toEqual(expectedAccounts[0]);
+      expect(accounts[1].toLowerCase()).toEqual(expectedAccounts[1]);
```

Comment:
Both of these produced weird results in the tests. The returned values from `service.eth.getAccounts` would result in the addresses having a mixture of upper and lower case characters whereas in the `testAccounts.json` where they were retrieved from, all addresses were lowercase. This may need to be investigated but is not a major issue

Reply:
No problem here. When the address is mixed-case, it's a "[checksum address](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md)"